packages/ai/features/augmentedPromptChat/ragChain.ts

```diff
@@ -21,10 +25,12 @@ Chat History:
 Follow Up Input: {question}
 Standalone question:`;
 const CONDENSE_QUESTION_PROMPT = PromptTemplate.fromTemplate(
-  condenseQuestionTemplate,
+  condenseQuestionTemplate
 );
 
-const answerTemplate = `Answer the question based only on the following context:
+const answerTemplate = `Answer the question based only on the following context, especially when asked to give an example.
```

Comment:
"based only on the following context"

Maybe be we could tell, "Answer the question leveraging the following context" or something alike